keras_cv/applications/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.applications.object_detection.retina_net.retina_net import RetinaNet
```

Comment:
do we want the public API to be all three at the same time? (a question more for API owner.)

keras_cv.application.RetinaNet
keras_cv.application.object_detection.RetinaNet
keras_cv.application.object_detection.retina_net.RetinaNet

Reply:
Removed other aliases.  If we need to support any more aliases, we can add them back

thanks!